server/controllers/events/events_controller_e2e_test.go

```diff
@@ -404,6 +404,44 @@ func TestGitHubWorkflow(t *testing.T) {
 				{"exp-output-merge.txt"},
 			},
 		},
+		{
```

Comment:
Another test worth adding is one where the resources are created using a count or for_each expression and the square brackets must be passed into the import command

For example, this may fail because locally in thebcommand line we would have to wrap the address in single quotes
```
atlantis import random_string.default["test"] id
```
Like this

```
atlantis import 'random_string.default["test"]' id
```

Please add the following when time permits

A test for count with or without single quotes
A test for for_each with or without single quotes
And documentation on the import comment

Reply:
Nice catch. The current implementation doesn't work both cases, cause step_runner uses `ctx.EscapedCommentArg` 😅 let me try to fix it.

Reply:
fix it and add test with for_each resource and -var option.

The bug is commandArgs and extraArgs order. terraform require `terraform import [options] ADDRESS ID`, so I need to append commandArgs after extraArgs.

https://github.com/runatlantis/atlantis/pull/2783/commits/656946e7f8d91b2fa59ffda12722baa48a3868d7